.githooks/pre-commit

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+LC_ALL=C
+
+local_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+valid_branch_regex="^(feature\/VIH-[0-9]{4})[a-zA-Z0-9._-]+$"
+
+message="There is something wrong with your branch name. Branch names in this project must start with feature/VIH-XXX. Your commit will be rejected. You should rename your branch to a valid name and try again."
```

Comment:
Please change this message also to say the exact format that is needed. i.e.  feature/VIH-XXX : Description

Reply:
I have change , but in the prepare-commit-msg, (pre-commit validate branch name and only prefix)